src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -237,7 +237,7 @@ properties:
 - they can be located either:
 
   - in ``<workflow-dir>/lib/python/``;
-  - or anywhere in your Python library path.
+  - or anywhere in your ``$CYLC_PYTHONPATH``.
```

Comment:
Is this one correct? Is `lib/python` added to `PYTHONPATH` or `CYLC_PYTHONPATH`?

Reply:
I think it should be what I've said. Will double check